recipes/fastcov/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: fastcov
+  version: 0.1.3
+source:
+  url: https://github.com/RaverJay/fastcov/archive/refs/tags/0.1.3.tar.gz
+  sha256: 7da7da416f4dc8c9dbbc7eb3d36938982813d620522e936ff56e5bc48289cdfe
+build:
+  number: '0'
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('fastcov', max_pin="x.x") }}
+requirements:
+  run:
+    - python
+    - matplotlib-base ==3.8.3
+    - pandas ==2.2.1
+    - pysam ==0.22.0
+    - seaborn ==0.13.2
```

Comment:
We prefer to use `=>` otherwise you can't package this in environments with other tools, can you confirm the tool absolutely requires those versions to execute?

Reply:
@jfy133 I changed the version requirements :) I dont think it needed them this strict, you are right with that.